src/mono/sample/Android/AndroidSampleApp.csproj

```diff
@@ -1,145 +1,38 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <RuntimeIdentifier>android-$(TargetArchitecture)</RuntimeIdentifier>
     <SelfContained>true</SelfContained>
     <PublishTrimmed>true</PublishTrimmed>
     <TrimMode>Link</TrimMode>
     <ForceAOT Condition="'$(ForceAOT)' == ''">false</ForceAOT>
+    <RunAOTCompilation Condition="'$(RunAOTCompilation)' == ''">$(ForceAOT)</RunAOTCompilation>
+    <StripDebugSymbols Condition="'$(Configuration)' == 'Release'">True</StripDebugSymbols>
+    <AppName>HelloAndroid</AppName>
+    <MainLibraryFileName>$(AssemblyName).dll</MainLibraryFileName>
+    <EnableDefaultAssembliesToBundle>true</EnableDefaultAssembliesToBundle>
+    <_UseAotDataFile>false</_UseAotDataFile>
```

Comment:
Why do we set unconditionally `_UseAotDataFile` to false?

Reply:
Just following what we had before: https://github.com/dotnet/runtime/blob/29013d8ae50f5bc35427a9155234ccebfa5e227c/src/mono/sample/Android/AndroidSampleApp.csproj#L106

Reply:
Added a clarifying comment